newm-server/src/main/resources/META-INF/services/io.ktor.server.config.ConfigLoader

```diff
@@ -0,0 +1,2 @@
+io.ktor.server.config.HoconConfigLoader
```

Comment:
Project is configured to use multiple loaders

Reply:
Not sure if it's useful or not, but there exists `MergedApplicationConfig`